ui/src/message_popup/popup_question.jsx

```diff
@@ -156,8 +156,9 @@ export default React.createClass({
   renderQuestion(){
     const {elapsedMs} = this.state;
     const {scaffolding} = this.props;
-    const {text, student, examples, nonExamples} = this.props.question;
+    const {text, students, examples, nonExamples} = this.props.question;
     const seconds = Math.round(elapsedMs / 1000);
+    const student = students === undefined ? undefined : students[0];
```

Comment:
If this is an array, let's change the questions so that when there are no students that's expressed explicitly as `studentIds: []` instead of the field just not being present.